src/Android/Xamarin.Android/packages.config

```diff
@@ -1,11 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Esri.ArcGISRuntime.Xamarin.Android" version="100.0.0" targetFramework="monoandroid60" />
+  <package id="Esri.ArcGISRuntime.Xamarin.Android" version="100.1.0" targetFramework="monoandroid60" />
   <package id="PCLCrypto" version="2.0.147" targetFramework="monoandroid60" />
-  <package id="PInvoke.BCrypt" version="0.3.152" targetFramework="monoandroid60" />
-  <package id="PInvoke.Kernel32" version="0.3.152" targetFramework="monoandroid60" />
-  <package id="PInvoke.NCrypt" version="0.3.152" targetFramework="monoandroid60" />
-  <package id="PInvoke.Windows.Core" version="0.3.152" targetFramework="monoandroid60" />
-  <package id="Validation" version="2.3.7" targetFramework="monoandroid60" />
-  <package id="Xamarin.Auth" version="1.3.1.1" targetFramework="monoandroid60" />
+  <package id="PInvoke.BCrypt" version="0.5.86" targetFramework="monoandroid60" />
+  <package id="PInvoke.Kernel32" version="0.5.86" targetFramework="monoandroid60" />
+  <package id="PInvoke.NCrypt" version="0.5.86" targetFramework="monoandroid60" />
+  <package id="PInvoke.Windows.Core" version="0.5.86" targetFramework="monoandroid60" />
+  <package id="Validation" version="2.4.15" targetFramework="monoandroid60" />
+  <package id="Xamarin.Android.Support.CustomTabs" version="23.3.0" targetFramework="monoandroid60" />
```

Comment:
Xamarin.Auth version 1.5.0.3 is dependent on it which is required for the author map sample